typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -223,6 +224,38 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     });
   }
 
+  updateProxyAdminOwnershipTxs(
+    actualConfig: Readonly<TokenRouterConfig>,
+    expectedConfig: Readonly<TokenRouterConfig>,
+  ): AnnotatedEV5Transaction[] {
+    const transactions: AnnotatedEV5Transaction[] = [];
+
+    // Return early because old warp config files did not have the
+    // proxyAdmin property
+    if (!expectedConfig.proxyAdmin) {
+      return transactions;
+    }
+
+    const actualProxyAdmin = actualConfig.proxyAdmin!;
+    assert(
+      actualProxyAdmin.address === expectedConfig.proxyAdmin.address,
```

Comment:
I can't because both address values might be undefined and `eqAddress` requires a string. I don't want to enforce that the addresses need to be defined because the old configuration files did not have those fields and changing them to required would be a breaking change

Reply:
i think eqAddress supports undefined addresses internally even if the type is misleading

Reply:
I'll test it and report back 🫡

Reply:
Seems to work fine!